test/server-spec.js

```diff
@@ -724,4 +730,3 @@ describe('Server manages requests in a set up repo', function () {
 
     });
 });
-
```

Comment:
Any way you could add a test with a successful non-participant licensing commitment?

Reply:
added in https://github.com/w3c/ash-nazg/pull/163/commits/e1670e7be4991492b9d1ed2b0ad263a312e038d1